addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxControlDimmer.java

```diff
@@ -73,7 +73,11 @@
      *         true if this control is suitable for this type
      */
     public static boolean accepts(String type) {
-        return type.equalsIgnoreCase(TYPE_NAME);
+        if (type != null) {
```

Comment:
This `null` check is not needed, because `TYPE_NAME`  is constant not `null` and `equalsIgnoreCase` is `null`-safe

Reply:
It seems it is rather for the possibility of 'type' argument being null. I
think the suggestion is generally correct and good, but change would need
to happen to all control classes, as it is repeated in them.
Instead I would suggest leave it as it is and wait for #2604 merge, in
which it will be ensured that no null value can be ever passed to these
methods.

Reply:
I might have been not clear in my previous comment: by wait for merge I don't suggest to delay merging of this PR. Just merge it as it is and later follow up with null annotations merge.